lib/elixir/lib/macro.ex

```diff
@@ -153,7 +153,7 @@ defmodule Macro do
     raise ArgumentError, bad_pipe(expr, call_args)
   end
 
-  # Without this, `Macro |> Env == Macro.Env`.
+  # Without this: "Macro |> Env == Macro.Env"
```

Comment:
Other than the quotes/backticks, please leave the trailing period in comments, as they're still sentences meant to be read by humans :)